.vscode/launch.json

```diff
@@ -5,14 +5,15 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "integration-tests - this test file",
+            "name": "integration-tests - individual",
             "request": "launch",
             "runtimeArgs": [
                 "run-script",
                 "start-tests",
                 "--",
                 "--runInBand",
                 "${relativeFile}",
```

Comment:
This was updated to run the current test that's open in VS Code - do we still need the 'individual' option here?

Reply:
No, I'll revert it